pkg/helpers/util.go

```diff
@@ -186,3 +187,36 @@ func Int2Str(ints []int32) []string {
 	}
 	return result
 }
+
+// IntList2Range takes a list of sorted integers and returns
+// string representation of its range
+// IntList2Range(0,1,2,3,4,5,6) -> 0-6
+// IntList2Range(12,13,14,15,16) -> 12-16
+func IntList2Range(ints []int32) string {
+	res := make([]int32, 2)
+	if len(ints) < 3 {
+		res = ints
+	} else {
+		res = []int32{ints[0], ints[len(ints)-1]}
```

Comment:
This will only work for consecutive integers. Maybe a better name would be `ConsecutiveIntsToRange()`. Also, would be nice to have it panic or return error if the numbers are not consecutive. Or maybe make it bulletproof by handling numbers in any order and with gaps :)